.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,19 @@
-<!--
 Thanks for submitting a PR, your contribution is really appreciated!
 
 Here is a quick checklist that should be present in PRs.
-(please delete this text from the final description, this is just a guideline)
--->
```

Comment:
As per https://github.com/pytest-dev/pytest/pull/6159#issuecomment-552139717 removing the comments might be good already.
Happy to create a PR, but please approve it here already, so that it does not get talked down again.

Reply:
I think the "(please delete...)" could be removed from here, but I'd leave the other instructions inside comments, otherwise you'll see them on many PRs.